failed to build

```
batch/batch.go:45:2: cannot find package "github-mirror/auth" in any of:
        /home/vtolstov/.go/src/github-mirror/auth (from $GOROOT)
        /home/vtolstov/go/src/github-mirror/auth (from $GOPATH)
batch/batch.go:46:2: cannot find package "github-mirror/mirror" in any of:
        /home/vtolstov/.go/src/github-mirror/mirror (from $GOROOT)
        /home/vtolstov/go/src/github-mirror/mirror (from $GOPATH)
```

```
mirror/readall.go:119: cannot use client.Repositories (type *github.RepositoriesService) as type repositoriesService in argument to fetchStatuses:
        *github.RepositoriesService does not implement repositoriesService (wrong type for ListStatuses method)
                have ListStatuses(string, string, string, *github.ListOptions) ([]*github.RepoStatus, *github.Response, error)
                want ListStatuses(string, string, string, *github.ListOptions) ([]github.RepoStatus, *github.Response, error)
mirror/readall.go:186: cannot use client.PullRequests (type *github.PullRequestsService) as type pullRequestsService in argument to fetchPullRequests:
        *github.PullRequestsService does not implement pullRequestsService (wrong type for List method)
                have List(string, string, *github.PullRequestListOptions) ([]*github.PullRequest, *github.Response, error)
                want List(string, string, *github.PullRequestListOptions) ([]github.PullRequest, *github.Response, error)
mirror/readall.go:192: cannot use client.PullRequests (type *github.PullRequestsService) as type pullRequestsService in argument to fetchComments:
        *github.PullRequestsService does not implement pullRequestsService (wrong type for List method)
                have List(string, string, *github.PullRequestListOptions) ([]*github.PullRequest, *github.Response, error)
                want List(string, string, *github.PullRequestListOptions) ([]github.PullRequest, *github.Response, error)
mirror/readall.go:192: cannot use client.Issues (type *github.IssuesService) as type issuesService in argument to fetchComments:
        *github.IssuesService does not implement issuesService (wrong type for ListComments method)
                have ListComments(string, string, int, *github.IssueListCommentsOptions) ([]*github.IssueComment, *github.Response, error)
                want ListComments(string, string, int, *github.IssueListCommentsOptions) ([]github.IssueComment, *github.Response, error)
```

Also does it possible to convert to go get-able repo?

--- a/app/operations.go
+++ b/app/operations.go
@@ -31,7 +31,7 @@ import (
 	"google.golang.org/appengine"
 	"google.golang.org/appengine/log"
 
-	"github-mirror/mirror"
+	"github.com/google/git-pull-request-mirror/mirror"
 )
 
 const (
@@ -423,7 +423,7 @@ func restartAbandonedOperations() {
 
 	repos, err := getAllRepoData(c)
 	if err != nil {
-		log.Errorf(c, "Can't load repos: %s", err.Error)
+		log.Errorf(c, "Can't load repos: %s", err.Error())
 		return
 	}
 

--- a/batch/batch.go
+++ b/batch/batch.go
@@ -42,8 +42,8 @@ import (
 	"github.com/google/git-appraise/repository"
 	"github.com/google/go-github/github"
 
-	"github-mirror/auth"
-	"github-mirror/mirror"
+	"github.com/google/git-pull-request-mirror/auth"
+	"github.com/google/git-pull-request-mirror/mirror"
 )
 
 var remoteRepository = flag.String("target", "", "Github repository to read data from")

--- a/mirror/conversions.go
+++ b/mirror/conversions.go
@@ -49,7 +49,7 @@ func ConvertTime(t time.Time) string {
 }
 
 // ConvertStatus converts a commit status fetched from the GitHub API into a CI report.
-func ConvertStatus(repoStatus github.RepoStatus) (*ci.Report, error) {
+func ConvertStatus(repoStatus *github.RepoStatus) (*ci.Report, error) {
 	result := ci.Report{}
 	if repoStatus.UpdatedAt != nil {
 		result.Timestamp = ConvertTime(*repoStatus.UpdatedAt)
@@ -80,7 +80,7 @@ func ConvertStatus(repoStatus github.RepoStatus) (*ci.Report, error) {
 }
 
 // ConvertPullRequest converts a pull request fetched from the GitHub API into a review request.
-func ConvertPullRequest(pr github.PullRequest) (*request.Request, error) {
+func ConvertPullRequest(pr *github.PullRequest) (*request.Request, error) {
 	if pr.Number == nil || pr.User.Login == nil ||
 		pr.Base == nil || pr.Base.Ref == nil || pr.Base.SHA == nil ||
 		(pr.CreatedAt == nil && pr.UpdatedAt == nil) {
@@ -121,7 +121,7 @@ func ConvertPullRequest(pr github.PullRequest) (*request.Request, error) {
 }
 
 // ConvertIssueComment converts a comment on the issue associated with a pull request into a git-appraise review comment.
-func ConvertIssueComment(issueComment github.IssueComment) (*comment.Comment, error) {
+func ConvertIssueComment(issueComment *github.IssueComment) (*comment.Comment, error) {
 	if issueComment.User == nil || issueComment.User.Login == nil || issueComment.Body == nil ||
 		(issueComment.UpdatedAt == nil && issueComment.CreatedAt == nil) {
 		return nil, ErrInsufficientInfo
@@ -144,7 +144,7 @@ func ConvertIssueComment(issueComment github.IssueComment) (*comment.Comment, er
 }
 
 // ConvertDiffComment converts a comment on the diff associated with a pull request into a git-appraise review comment.
-func ConvertDiffComment(diffComment github.PullRequestComment) (*comment.Comment, error) {
+func ConvertDiffComment(diffComment *github.PullRequestComment) (*comment.Comment, error) {
 	if diffComment.User == nil || diffComment.User.Login == nil || diffComment.Body == nil ||
 		(diffComment.UpdatedAt == nil && diffComment.CreatedAt == nil) ||
 		diffComment.OriginalCommitID == nil {
@@ -190,7 +190,7 @@ func ConvertDiffComment(diffComment github.PullRequestComment) (*comment.Comment
 //
 // This method requires a local clone of the repository in order to compute the locations of
 // the different commits in the review.
-func ConvertPullRequestToReview(pr github.PullRequest, issueComments []github.IssueComment, diffComments []github.PullRequestComment, repo repository.Repo) (*review.Review, error) {
+func ConvertPullRequestToReview(pr *github.PullRequest, issueComments []*github.IssueComment, diffComments []*github.PullRequestComment, repo repository.Repo) (*review.Review, error) {
 	request, err := ConvertPullRequest(pr)
 	if err != nil {
 		return nil, err
@@ -242,7 +242,7 @@ func ConvertPullRequestToReview(pr github.PullRequest, issueComments []github.Is
 }
 
 // commentStartLine takes a PullRequestComment and returns the comment's start line.
-func commentStartLine(diffComment github.PullRequestComment) (uint32, error) {
+func commentStartLine(diffComment *github.PullRequestComment) (uint32, error) {
 	// This takes some contortions to figure out. The diffComment has a "position"
 	// field, but that is not the position of the comment. Instead, that is the
 	// position of the comment within the diff. Furthermore, this diff is a unified diff,
@@ -288,7 +288,7 @@ func commentStartLine(diffComment github.PullRequestComment) (uint32, error) {
 }
 
 // computeReviewStartingCommit computes the first commit in the review.
-func computeReviewStartingCommit(pr github.PullRequest, repo repository.Repo) (string, error) {
+func computeReviewStartingCommit(pr *github.PullRequest, repo repository.Repo) (string, error) {
 	if pr.Base == nil || pr.Base.SHA == nil ||
 		pr.Head == nil || pr.Head.SHA == nil {
 		return "", ErrInsufficientInfo

--- a/mirror/readall.go
+++ b/mirror/readall.go
@@ -43,16 +43,16 @@ var (
 
 // Can be stubbed out in testing; satisfied by github.Client.Repositories
 type repositoriesService interface {
-	ListStatuses(owner, repo, ref string, opt *github.ListOptions) ([]github.RepoStatus, *github.Response, error)
+	ListStatuses(owner, repo, ref string, opt *github.ListOptions) ([]*github.RepoStatus, *github.Response, error)
 }
 
 type pullRequestsService interface {
-	List(owner string, repo string, opt *github.PullRequestListOptions) ([]github.PullRequest, *github.Response, error)
-	ListComments(owner string, repo string, number int, opt *github.PullRequestListCommentsOptions) ([]github.PullRequestComment, *github.Response, error)
+	List(owner string, repo string, opt *github.PullRequestListOptions) ([]*github.PullRequest, *github.Response, error)
+	ListComments(owner string, repo string, number int, opt *github.PullRequestListCommentsOptions) ([]*github.PullRequestComment, *github.Response, error)
 }
 
 type issuesService interface {
-	ListComments(owner string, repo string, number int, opt *github.IssueListCommentsOptions) ([]github.IssueComment, *github.Response, error)
+	ListComments(owner string, repo string, number int, opt *github.IssueListCommentsOptions) ([]*github.IssueComment, *github.Response, error)
 }
 
 type retryableRequest func() (*github.Response, error)
@@ -204,8 +204,8 @@ func GetAllPullRequests(local repository.Repo, remoteUser, remoteRepo string, cl
 	return output, nil
 }
 
-func fetchPullRequests(remoteUser, remoteRepo string, prs pullRequestsService) ([]github.PullRequest, error) {
-	var results []github.PullRequest
+func fetchPullRequests(remoteUser, remoteRepo string, prs pullRequestsService) ([]*github.PullRequest, error) {
+	var results []*github.PullRequest
 	err := executeListRequest(func(listOpts github.ListOptions) (*github.Response, error) {
 		opts := &github.PullRequestListOptions{
 			State:       "all",
@@ -224,8 +224,8 @@ func fetchPullRequests(remoteUser, remoteRepo string, prs pullRequestsService) (
 }
 
 // fetchComments fetches all of the comments for each issue it gets and then converts them.
-func fetchComments(pr github.PullRequest, remoteUser, remoteRepo string, prs pullRequestsService, is issuesService) ([]github.IssueComment, []github.PullRequestComment, error) {
-	var issueComments []github.IssueComment
+func fetchComments(pr *github.PullRequest, remoteUser, remoteRepo string, prs pullRequestsService, is issuesService) ([]*github.IssueComment, []*github.PullRequestComment, error) {
+	var issueComments []*github.IssueComment
 	err := executeListRequest(func(listOpts github.ListOptions) (*github.Response, error) {
 		listOptions := &github.IssueListCommentsOptions{
 			ListOptions: listOpts,
@@ -239,7 +239,7 @@ func fetchComments(pr github.PullRequest, remoteUser, remoteRepo string, prs pul
 	if err != nil {
 		return nil, nil, err
 	}
-	var diffComments []github.PullRequestComment
+	var diffComments []*github.PullRequestComment
 	err = executeListRequest(func(listOpts github.ListOptions) (*github.Response, error) {
 		listOptions := &github.PullRequestListCommentsOptions{
 			ListOptions: listOpts,

--- a/mirror/readall_test.go
+++ b/mirror/readall_test.go
@@ -63,7 +63,7 @@ type repoServiceStub struct {
 	Responses []repoServiceResponse
 }
 
-func (s *repoServiceStub) ListStatuses(owner, repo, ref string, opt *github.ListOptions) ([]github.RepoStatus, *github.Response, error) {
+func (s *repoServiceStub) ListStatuses(owner, repo, ref string, opt *github.ListOptions) ([]*github.RepoStatus, *github.Response, error) {
 	if s.Index >= len(s.Responses) {
 	}
 	r := s.Responses[s.Index]
@@ -102,7 +102,7 @@ func TestFetchReports(t *testing.T) {
 		expectedReports = append(expectedReports, *successReport)
 		expectedReports = append(expectedReports, *failureReport)
 		responses = append(responses, repoServiceResponse{
-			Results: []github.RepoStatus{
+			Results: []*github.RepoStatus{
 				successResult,
 				failureResult,
 			},

